Tasks/ManualIntervention/task.json

```diff
@@ -30,6 +30,14 @@
                 "rows": "10",
                 "maxLength": "500"
             }
+        },
+        {
+            "name": "emailRecipients",
+            "type": "identities",
```

Comment:
Yes. This is new type. Using common identity picker, i hope it will support IDN mail ids. I will check with team.